Doc/library/asyncio-eventloop.rst

```diff
@@ -188,19 +188,21 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`ThreadPoolExecutor`. After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the amount of time this method gives the
+   executor to finish joining. The default value is ``None``, which means the
+   method allows the executor an unlimited amount of time.
 
-   If the timeout duration is reached, a warning is emitted and executor is
-   terminated without waiting for its threads to finish joining.
+   If the timeout duration is reached, this method emits a warning and
+   terminates the default executor without waiting for its threads to finish
+   joining.
 
-   Note that there is no need to call this function when
-   :func:`asyncio.run` is used.
+   Note that there is no need to call this function when :func:`asyncio.run` is
+   used, because that high-level function handles default executor shutdown
```

Comment:
One spot where I think more explanation would be beneficial: explaining *why* "there is no need to call this function when `asyncio.run()` is used." Please correct my guess at the "why", if needed.

Reply:
@gvanrossum Is this "why" accurate?

Reply:
Yeah, it's just because `asyncio.run()` calls various shutdown methods.